draft-ietf-webpush-vapid.md

```diff
@@ -158,18 +168,54 @@ This information would be repeated with each request, but that overhead is
 greatly reduced by header compression [RFC7541] in HTTP/2 [RFC7540].
 
 
-### Contact Information Header Field
+## Client-Vended Tokens {#client-token}
+
+In this model, clients generates a token that it uses to prove ownership over a
+private key.  Use of the same key over time establishes a continuous identity.
+
+Push message requests can be accompanied by a JSON Web Token (JWT) [RFC7519].
+This token is not bound to the request.  As a result, the token is reusable over
+a limited time, based on the value of the "exp" (Expiry) claim in the token.  An
+"exp" claim MUST be included; similarly, an "aud" (Audience) claim MUST include
+the string "webpush".
```

Comment:
Should the Audience instead be the domain of the push endpoint URL to mitigate stolen token re-use across Push providers?